server/src/uuid/model/taxonomy_term/messages.rs

```diff
@@ -103,3 +109,35 @@ pub mod taxonomy_term_move_mutation {
         }
     }
 }
+
+pub mod taxonomy_term_create_mutation {
+    use super::*;
+    use crate::instance::Instance;
+    use crate::uuid::Uuid;
+    use crate::vocabulary::model::TaxonomyType;
```

Comment:
shouldn't this be moved to this modul (like in `model.rs`)

Reply:
I'm not sure if I undestood... which model.rs?

Reply:
Move enum `TaxonomyType` into `server/src/uuid/model/taxonomy_term`